### ISSUE TITLE
LUA "system tool" icon does not scale to adjust for LCD resolution

### ISSUE THREAD

**Issue by azaz44** [FrSkyRC/ETHOS-Feedback-Community#2247]
This is how it looks on X20 (which is correct)

![image](https://user-images.githubusercontent.com/42830563/209660445-efe3a657-1ede-4e1f-a44c-6ca7835c13f7.png)

This is how it looks on X10 (not scaled properly)

![image](https://user-images.githubusercontent.com/42830563/209660523-61853331-29a6-4052-af24-a42a0440b410.png)

**Comment by azaz44** [comment]
@bsongis 
We had discussions here: #2284 and here: #2283. It looks like we need different icons for different screen resolutions.

Then I wonder how to solve this issue here. I tried this:

```lua
function init()
  local w, h = lcd.getWindowSize()
  print("w=", w, " h=", h)
  system.registerSystemTool(...)
end

return {init=init}
```

But `w` and `h` are `nil`. Which is expected, as we don't have any window yet.
But we need something to tell us, to load the right icons for this radio.
Is there any way to figure out LCD size?

**Comment by azaz44** [comment]
@bsongis-frsky 
I wondered, if as a quick solution / workaround, we could have lcd size included here:

![image](https://user-images.githubusercontent.com/42830563/213006195-dc997e44-3e4e-48a7-a435-3a2468280fe1.png)

for example as `lcdWidth` and `lcdHeight`. This would still require scripts to have multiple icon versions and provide the right one, but at least we would have some chance to select the right icon.

As it is now, `lcd.getWindowSize` does not return anything useful when script is in `init()` and I don't see any other way to load and return the right icon. Could try to select it by `getVersion().boardName` but this would be really ugly..

**Comment by azaz44** [comment]
@bsongis-frsky It looks like it was closed and completed, but not sure if correct. There are `lcdWidth` and `lcdHeight` available if I test version `1.4.6` but instead of sizes they both contain `true`. This is what I get in simulator:
```
suffix:
lcdHeight: true
minor: 4
revision: 6
version: 1.4.6
major: 1
board: X18S
simulation: true
lcdWidth: true
```

**Comment by bsongis-frsky** [comment]
Right. Typo fixed for 1.4.7
